server/config/persist_options.go

```diff
@@ -50,11 +50,12 @@ type PersistOptions struct {
 	replicationMode atomic.Value
 	labelProperty   atomic.Value
 	clusterVersion  unsafe.Pointer
+	cfg             *Config
```

Comment:
How about adding an interface function into Cluster?

Reply:
The most values of the  persistOptions is transferred by config and I think the cluster should export one method to manage all config.

Reply:
Those configs in persist option can be changed dynamically, I prefer not to put `cfg` inside it.